parse/parse.d.ts

```diff
@@ -72,14 +72,16 @@ declare namespace Parse {
         then<U>(resolvedCallback: (value: T) => U, rejectedCallback?: (reason: any) => U): IPromise<U>;
     }
 
-    interface Promise<T> {
+    class Promise<T> {
+
+        static as(resolvedValue: any): Promise<any>;
+        static error(error: any): Promise<any>;
```

Comment:
I would think that these should take a type parameter `T` and return a `Promise<T>`